packages/design-system/src/test-data/cookieMockData.ts

```diff
@@ -16,8 +16,7 @@
 /**
  * External dependencies.
  */
-import { type Cookie as ParsedCookie } from 'simple-cookie';
-import { TabCookies, TabFrames } from '@google-psat/common';
+import { CookieData, TabCookies, TabFrames } from '@google-psat/common';
```

Comment:
Please consider using the keyword `type` for these imports.

Reply:
Added in latest commits